### PR TITLE
Add brief explaination of Jeong Woon

### PR DESCRIPTION
**Pull request by isyangban**
Hello, I made my own page as described in helpdesk [link](https://github.com/kaist-cp/helpdesk#first-things-to-do).
Also I tried to make my mattermost account but it didn't work well. Can I get some help on making my mattermost account?

### CODE REVIEW

**Comment by jeehoonkang** [comment]
@isyangban please rebase so that I can merge.

**Comment by isyangban** [comment]
I rebased to the latest master commit.

File: _includes/person_experience.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{%- assign person_id = include.person_id %}
```
Comment (jeehoonkang) [inline _includes/person_experience.md:1]: I'd rather put those in the personal page, as I guess people have different way of writing up their own experiences.

**Comment by jeehoonkang** [comment]
@isyangban ping

File: _includes/person_experience.md (hunk 1 above)
Comment (isyangban) [reply _includes/person_experience.md]: Ok, fixed.

**Pull request merged**